py3status/modules/README.md

```diff
@@ -159,6 +159,7 @@ also waiting.
 Configuration parameters:
   - `cache_timeout` How often we refresh this module in seconds *(default 600)*
   - `format` Display format to use
+  - `hide_if_zero` Don't show on bar if True *(default False)*
```

Comment:
These should be in alphabetical order, you've also put it in the middle of the format (before the default).

This file gets auto generated before a release but there is no harm in you adding the updates.